src/ctap/response.rs

```diff
@@ -228,6 +243,11 @@ mod test {
             options: None,
             max_msg_size: None,
             pin_protocols: None,
+            max_credential_count_in_list: None,
+            max_credential_id_length: None,
+            transports: None,
+            algorithms: None,
+            firmware_version: None,
```

Comment:
IIRC a `None` option is not serialized in the CBOR map. It would be useful to have tests with `Some(dummy_value)` as well, otherwise we miss testing coverage in the parsing side.

Reply:
Test added.